src/cacheRefresher.js

```diff
@@ -27,11 +27,12 @@ governing permissions and limitations under the License.
    * @param {ConnectionParameters} connectionParameters used to created the client provide as parameter
    * @param {string} rootKey is an optional root key to use for the storage object
    */
-    constructor(cache, client, connectionParameters, rootKey) {
+    constructor(cache, client, connectionParameters, rootKey, refreshedtype) {
```

Comment:
maybe not "refreshedtype" but "cacheSchema"? can this be a list? or for one schema, we have one cacheRefresher?

Reply:
We can use a list in the future, but for the moment the cache  `_entityCache` contains only data for `xtk:schema` https://github.com/adobe/acc-js-sdk/blob/master/src/client.js#L1102.
We have one CacheRefresher per Cache, so currently in `Client` class we have two cacheRefreshers one for `_entityCache` and one for `_optionCache`